modules/core/src/materials/phong-material.js

```diff
@@ -0,0 +1,13 @@
+const defaultProps = {
+  ambient: 0.4,
+  diffuse: 0.6,
+  shininess: 32,
+  specularColor: [255, 255, 255]
+};
+
+export default class PhongMaterial {
```

Comment:
I recommend creating a base Material class that we can derive `PhongMaterial` and `PBRMaterial` from.

Assume that the idea is that the selection of material subclass effectively selects the shader module used to render the material?